### PR TITLE
v2024.02.2 RC

### PR DESCRIPTION
**Pull request by adayton1**
This PR is the release candidate for the v2024.02.2 release.

Note: The companion PR that only shows changes between the RC branch and develop is https://github.com/LLNL/CHAI/pull/268. Please review that PR and approve this one to merge.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -11,7 +11,7 @@ cmake_policy(SET CMP0025 NEW)
 include(CMakeDependentOption)
 include(CMakePackageConfigHelpers)
 
-project(Chai LANGUAGES C CXX VERSION 2024.02.1)
+project(Chai LANGUAGES C CXX VERSION 2024.02.2)
```
Comment (rhornung67) [inline CMakeLists.txt:14]: new version should be 2024.07.0

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
 [comment]: # (# SPDX-License-Identifier: BSD-3-Clause)
 [comment]: # (#################################################################)
 
-# CHAI v2024.02.1
+# CHAI v2024.02.2
```
Comment (rhornung67) [inline README.md:9]: v2024.07.0

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -13,6 +13,12 @@ in this file.
 
 The format of this file is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [Version 2024.02.2] - Release date 2024-06-26
```
Comment (rhornung67) [inline RELEASE_NOTES.md:16]:
```suggestion
## [Version 2024.07.0] - Release date 2024-06-26
```

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -13,6 +13,12 @@ in this file.
...
+
+### Changed
+- Updated to RAJA v2024.02.2
```
Comment (rhornung67) [inline RELEASE_NOTES.md:19]: You'll want to update to RAJA v2024.07.0 when released.

File: CMakeLists.txt (hunk 1 above)
Comment (adayton1) [reply CMakeLists.txt:14]: I'm doing a 2024.02.2 release first so that there is a version of CHAI that works with RAJA 2024.02.2.

**Review by robinson96 — APPROVED** [review]
LGTM

**Review by liu15 — APPROVED** [review]

**Pull request merged**